vNext/test/AWSUtils.Tests/AWSUtils.Tests.csproj

```diff
@@ -17,15 +21,17 @@
     <Reference Include="System.Web.Extensions" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
+
   <ItemGroup>
     <PackageReference Include="AWSSDK.DynamoDBv2" Version="3.3.3" />
     <PackageReference Include="AWSSDK.SQS" Version="3.3.1.7" />
     <PackageReference Include="FluentAssertions" Version="4.19.0" />
     <PackageReference Include="xunit" Version="2.2.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
     <PackageReference Include="xunit.runner.utility" Version="2.2.0" />
-    <PackageReference Include="Xunit.SkippableFact" Version="1.2.14" />
+    <PackageReference Include="Xunit.SkippableFact" Version="1.3.1" />
```

Comment:
Let's get rid of the xunit stuff in the future, since they should transitively come from TestExtensions. It's fine to do right now, is so that we remember